src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/BrowseSolutions/ViewASolution.cs

```diff
@@ -164,7 +163,9 @@ public void ThenTheDownloadMoreInformationButtonDownloadsAFile(string fileFormat
         public void WhenTheLastUpdatedValueInTheSolutionTableIsUpdated(string tableName)
         {
             var updatedDate = DateTime.Now;
-            expectedLastUpdatedDate = updatedDate.ToString("yyyy-MM-dd");
+
+            // Use long variant of date (i.e. 12 December 2019)
+            expectedLastUpdatedDate = updatedDate.ToString("dd MMMM yyyy");
```

Comment:
use the ConvertDateToLongDateTime method

Reply:
That adds the time onto the end of the date, which is not correct for the scenario under test